src/main/resources/db/migration/V20251002.2379__update_yksilo_luvat.sql

```diff
@@ -0,0 +1,4 @@
+UPDATE yksilo
+  SET lupa_kayttaa_tekoalyn_koulutukseen = lupa_luovuttaa_tiedot_ulkopuoliselle, 
+    lupa_luovuttaa_tiedot_ulkopuoliselle = lupa_kayttaa_tekoalyn_koulutukseen
+  WHERE (lupa_kayttaa_tekoalyn_koulutukseen IN (FALSE, TRUE) OR lupa_luovuttaa_tiedot_ulkopuoliselle IN (FALSE, TRUE));
```

Comment:
WHERE clause does not do any harm, but I'm just wondering if it is needed at all? Why can't we update all rows?